src/player/PlayerComponent.cpp

```diff
@@ -1122,7 +1122,7 @@ void PlayerComponent::setAudioConfiguration()
   }
   else
   {
-    mpv::qt::command(m_mpv, QStringList() << "af" << "del" << "@ac3");
+    mpv::qt::command(m_mpv, QStringList() << "af" << "remove" << "@ac3");
```

Comment:
```
2023-06-07 22:06:47.349 [warning] PlayerComponent::handleMpvEvent @ 594 - cplayer: Option af-del: -del is deprecated! Use -remove (removes by content instead of by index).
2023-06-07 22:06:47.349 [warning] PlayerComponent::handleMpvEvent @ 594 - cplayer: Option af-del: item label @ac3 not found.
```
With remove instead of del, you're apparently supposed to pass the item itself instead of an index. Not sure how to do that here, as I don't know the mpv command syntax.